Packages/com.unity.inputsystem/InputSystem/Editor/Internal/InputControlTreeView.cs

```diff
@@ -184,30 +189,18 @@ private ControlItem BuildControlTreeRecursive(InputControl control, int depth, r
             var offset = controlOffset - rootOffset;
 
             // Read state.
-            GUIContent value;
-            GUIContent[] values;
-            if (!ReadState(control, out value, out values))
-                return null;
-
-            ////TODO: come up with nice icons depicting different control types
-
             var item = new ControlItem
             {
                 id = id++,
-                displayName = control.name,
                 control = control,
                 depth = depth,
-                layout = new GUIContent(control.layout),
-                format = new GUIContent(control.stateBlock.format.ToString()),
-                offset = new GUIContent(offset.ToString()),
-                bit = new GUIContent(control.stateBlock.bitOffset.ToString()),
-                sizeInBits = new GUIContent(control.stateBlock.sizeInBits.ToString()),
-                type = new GUIContent(control.GetType().Name),
-                value = value,
-                values = values,
                 children = children
             };
 
+            ////TODO: come up with nice icons depicting different control types
```

Comment:
Was this TODO added as part of this work? Some associated task that should be filed? Feels unrelated?

Reply:
It's unrelated, just moved code around